spec/integration/real_api/a_real_api_spec.rb

```diff
@@ -0,0 +1,49 @@
+describe 'Mounting a real API' do
+  let!(:server_thread) do
+    Thread.new do
+      `fuser -k 9292/tcp  2>/dev/null`
+      system("rackup spec/integration/real_api/example_api/config.ru 2>/dev/null")
```

Comment:
I meant that this call will fail without an exception if, for example, `rackup` cannot run. You need to handle the return from it.

Reply:
:+1: